<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>1064</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;学号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="birth">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;生日：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="birth_line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>450</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>450</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>450</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="shearch">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insert">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>650</y>
     <width>781</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
